flexget/components/jellyfin/from_jellyfin.py

```diff
@@ -0,0 +1,89 @@
+from loguru import logger
+
+from flexget.components.jellyfin.api_jellyfin import JellyfinAuth, JellyfinApi
+from flexget.config_schema import one_or_more
+from flexget import plugin
+from flexget.event import event
+from flexget.components.jellyfin.jellyfin_util import SCHEMA_SERVER_TAG, SORT_FIELDS
+
+logger = logger.bind(name='from_jellyfin')
+
+
+class JellyfinInput:
+    """
+    Returns Jellyfin Inputs
+
+    Example:
+        from_jellyfin:
+            server:
+                host: http://localhost:8096
+                username: <username>
+                apikey: <apikey>
+                return_host: wan
+            list: TV
+            types: series
+    """
+
+    auth = None
+
+    schema = {
+        'type': 'object',
+        'properties': {
+            **SCHEMA_SERVER_TAG,
+            'list': {'type': 'string'},
+            'types': one_or_more(
+                {'type': 'string', 'enum': ['movie', 'series', 'season', 'episode']}
+            ),
+            'watched': {'type': 'boolean'},
+            'favorite': {'type': 'boolean'},
+            'sort': {
+                'oneOf': [
+                    {
+                        'type': 'string',
+                        'enum': SORT_FIELDS,
+                    },
+                    {
+                        'type': 'object',
+                        'properties': {
+                            'field': {
+                                'type': 'string',
+                                'enum': SORT_FIELDS,
+                            },
+                            'order': {'type': 'string', 'enum': ['ascending', 'descending']},
+                        },
+                        'required': ['field', 'order'],
+                    },
+                ]
+            },
+        },
+        'required': ['server', 'list'],
+        'additionalProperties': False,
+    }
+
+    def login(self, config):
+        if self.auth and self.auth.logged:
+            return
+
+        if not isinstance(config, dict):
+            config = {}
+
+        self.auth = JellyfinAuth(**config)
```

Comment:
This is a bit bad practice as it is not very explicit and will be problematic if config does no longer match the constructor signature (ie. adding keys).

Reply:
I have used Emby plug-in and made changes so that it will work with Jellyfin :), I will wait for the changes in emby code which is using the same (since I am not actually a coder).
>if not isinstance(config, dict):
config = {}
self.auth = EmbyAuth(**config)
self.auth.login(True)